CMakeLists.txt

```diff
@@ -83,4 +83,4 @@ configure_file(${MPI_CMAKE_MODULES_SCRIPTS_DIR}/mpi_clang_format
 ##########################
 # building documentation #
 ##########################
-build_doxygen_documentation()
\ No newline at end of file
+build_documentation()
```

Comment:
It would probably fit better to rename this to something like `add_documentation` how, as it is not actively building, just adding the target.  But this would require renaming in a lot of packages, so maybe not worth the effort.

Reply:
anyway we will have to adapt all the CMakeLists.txt. So I do not mind calling it `add_documentation`
@vincentberenz what do you think?

Reply:
Oo so we would also have a `add_doxygen_documentation` and a `add_sphinx_documentation`?

Reply:
no real preference on my side